modules/apps/fragment/fragment-impl/src/main/java/com/liferay/fragment/internal/processor/PortletRegistryImpl.java

```diff
@@ -233,6 +243,9 @@ protected void unsetPortlet(
 	private static final Log _log = LogFactoryUtil.getLog(
 		PortletRegistryImpl.class);
 
+	private final Map<String, String> _aliasPortletNames =
```

Comment:
@W2jx Please make this map store portletName -> alias, the reason is we know portlet name should be unique, but you are not sure if alias is unique or not, though the final result for getPortletAliases should be same, but if someone happens to use your map later, then this hidden information might be ignored and cause issue.
So to be safe, we should make the map store all the information to avoid possible mistakes in the future.

Reply:
Hi @tinatian, In our code, alias is unique and other classes are using alias to get portletName.